ledger/store/trackerdb/sqlitedriver/schema.go

```diff
@@ -948,11 +948,23 @@ func accountsAddCreatableTypeColumn(ctx context.Context, e db.Executable, popula
 	var creatableTypeOnResourcesRun bool
 	err := e.QueryRow("SELECT 1 FROM pragma_table_info('resources') WHERE name='ctype'").Scan(&creatableTypeOnResourcesRun)
 	if err == nil {
-		// Already exists.
+		// Check if any ctypes are invalid
+		var count uint64
+		err0 := e.QueryRow("SELECT COUNT(*) FROM resources WHERE ctype NOT IN (0, 1)").Scan(&count)
```

Comment:
nit: `= -1` probably faster

Reply:
I was checking for invalid values, of which -1 is amongst them.